framework/src/org/checkerframework/framework/qual/TypeQualifiers.java

```diff
@@ -6,68 +6,148 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
-
-import javax.annotation.processing.SupportedAnnotationTypes;
-
 import org.checkerframework.framework.type.AnnotatedTypeFactory;
-import org.checkerframework.framework.type.AnnotatedTypeMirror;
 import org.checkerframework.qualframework.base.Checker;
 
 /**
- * An annotation that lists the type qualifiers supported by the annotated
- * {@code Checker}.
+ * This meta-annotation is deprecated.
```

Comment:
These examples are great. But instead of putting them on the deprecated annotations, should you move them to the createSupportedTypeQualifiers method?